libazureinit/src/config.rs

```diff
@@ -307,21 +307,27 @@ impl Config {
             if cli_path.is_dir() {
                 figment = Self::merge_toml_directory(figment, cli_path)?;
             } else {
-                tracing::info!(
```

Comment:
In my opinion this is nice to have at info-level since configuration being unexpected included/not included is a thing that happens regularly.  This applies to all the log statements about files being included.